pkgs/top-level/all-packages.nix

```diff
@@ -33223,7 +33223,7 @@ with pkgs;
   shaka-packager = callPackage ../applications/video/shaka-packager { };
 
   # Wraps without triggering a rebuild
-  wrapMpv = callPackage ../applications/video/mpv/wrapper.nix { };
+  inherit (mpv-unwrapped) wrapMpv;
```

Comment:
Why not move this to aliases.nix?

Reply:
`aliases.nix` triggers ofBorg banishing mechanisms.